### PR TITLE
Add Language Urdu

### LINKED ISSUE #3685: Adding a new language (Urdu)
I want to add and translate English to Urdu language. Some information about the Urdu language is;

English Name: Urdu
Native Name: Urdu

Type: language
Subtag: ur
Description: Urdu
Suppress-Script: Arab

### PR DESCRIPTION
**Pull request by bramkragten**
Closes https://github.com/home-assistant/home-assistant-polymer/issues/3685

### CODE REVIEW

File: src/translations/translationMetadata.json [hunk 1]
```diff
@@ -151,6 +151,9 @@
   "uk": {
     "nativeName": "Українська"
   },
+  "ur": {
+    "nativeName": "Urdu"
```
Comment (balloob) [inline src/translations/translationMetadata.json]: We generally write the language in the native script of that language, which would be اُردُو

CC @msanaullahsahar is that correct?

Comment (msanaullahsahar) [reply src/translations/translationMetadata.json]: Yes, اُردُو is the correct native name for URDU.

**Comment by bramkragten** [comment]
@msanaullahsahar I also added it to Lokalize, so you can start translating! :-)

**Comment by msanaullahsahar** [comment]
Thanks for it. However i couldn't see URDU language there. Do i need to wait for it to appear there ..

On 19/09/2019 9:10:50 PM, Bram Kragten <notifications@github.com> wrote:

@msanaullahsahar<https://github.com/msanaullahsahar> I also added it to Lokalize, so you can start translating! :-)

**Comment by bramkragten** [comment]
Can you check it again, you should see it now

**Comment by balloob** [comment]
When we add a language, we need to give people permissions for it too on Lokalise.

**Review by balloob — APPROVED** [review]

**Comment by bramkragten** [comment]
Is there a way to give everyone permission to a language at once? Or is it a manual job?

**Comment by balloob** [comment]
Manual :(

**Pull request merged**